### ISSUE TITLE
add data input for Kalman filter integration

### ISSUE THREAD

**Issue by Hagar-Usama** [Das-Auto/SFND_Lidar_Obstacle_Detection#1]

**Comment by Hagar-Usama** [comment]
@youssefMHanna  @michaelramez 
Please check #2 :pray:

**Comment by youssefMHanna** [comment]
Great thanks no need to pickle
you can do instead a function similar to the TTC requested one (spin_once) that runs one frame
which returns float[number_of_boxes_per_frame][6]
// the 6 being box.x_min , box.y_min , box.z_min , box.x_max , box.y_max , box.z_max

**Comment by Hagar-Usama** [comment]
> you can do instead a function similar to the TTC requested one (spin_once) that runs one frame
> which returns float[number_of_boxes_per_frame][6]

That is another task (for ROS integration) as mentioned at the last meeting? Michael is no longer blocked by me, right? @youssefMHanna

**Comment by youssefMHanna** [comment]
> > you can do instead a function similar to the TTC requested one (spin_once) that runs one frame
> > which returns float[number_of_boxes_per_frame][6]
> 
> That is another task (for ROS integration) as mentioned at the last meeting? Michael is no longer blocked by me, right? @youssefMHanna 
> 

Yes Michael has already started this request is part of the ros integration task

**Comment by Hagar-Usama** [comment]
> Yes Michael has already started, this request is part of the ros integration task

Alright, I will create another issue and assign it to me :v: .
